Makefile

```diff
@@ -13,7 +13,7 @@ include Makefile.global
 all: extension
 
 # build extension
-extension:
+extension: src/include/citus_version.h
```

Comment:
Shouldn't the be using a variable rather than a full relative path? Aren't there various `src` variables we might want to use instead?